airflow/providers/amazon/aws/triggers/eks.py

```diff
@@ -27,6 +28,185 @@
 from airflow.triggers.base import BaseTrigger, TriggerEvent
 
 
+class EksCreateClusterTrigger(BaseTrigger):
+    """
+    Trigger for EksCreateClusterOperator.
+    The trigger will asynchronously wait for the cluster to be created.
```

Comment:
```suggestion
    Trigger for EksCreateClusterOperator.
    
    The trigger will asynchronously wait for the cluster to be created.
```

I'm working on enabling D205 style checks which requires a space after the first "summary" line, if you don't mind.